lib/common/common/ByteSwap.h

```diff
@@ -50,6 +48,22 @@ RtlUlonglongByteSwap(
     _In_ ULONGLONG Source
     );
 #endif
-#endif
 
+#elif defined(__APPLE__)
```

Comment:
it's predefined by GCC and Clang, and probably older Apple toolchains too (eg https://github.com/llvm-mirror/clang/blob/ea88b232d6f5178898909ecd39688b5c675d39ab/lib/Basic/Targets.cpp#L115)